recipes/tmtk/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "tmtk" %}
+{% set version = "0.4.4.dev3" %}
+{% set sha256 = "d94600bd60b120ae96ef9e6fad5019114836bf6da0bc181113ab7433a2d04f75" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<34]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - python
+    - pandas
+    - numpy 1.11.*
+    - tqdm
+    - arrow
+    - click
+    - jupyter
+    - xlrd
+    - notebook
+    - requests
+    - ipython
+    - widgetsnbextension
+
+  run:
+    - python
+    - pandas
+    - numpy >=1.11
+    - tqdm
+    - arrow
+    - click
+    - jupyter
+    - xlrd
+    - notebook
+    - requests
+    - ipython
+    - widgetsnbextension
+
+test:
+  imports:
+    - tmtk
+
+about:
+  home: http://github.com/thehyve/tmtk
+  license: GPL-3.0
```

Comment:
It looks like it might be `LGPL-3.0` rather than `GPL-3.0`:

https://github.com/thehyve/tmtk/blob/master/LICENSE.txt#L1